src/state/mod.rs

```diff
@@ -106,14 +101,10 @@ impl IndexerState {
     ) -> anyhow::Result<ExtensionType> {
         // prune the root branch
         if let Some(k) = self.transition_frontier_length {
-            if let Some(interval) = self.prune_interval {
-                if self.blocks_processed % interval == 0 {
-                    info!("pruning transition frontier at k={}", k);
-                    self.root_branch.prune_transition_frontier(k);
-                }
-            } else {
-                info!("pruning transition frontier at k={}", k);
-                self.root_branch.prune_transition_frontier(k);
+            if self.root_branch.height() > 5 * k as usize {
```

Comment:
can we make this constant configurable?

Reply:
I think that's a good idea!

Reply:
Will include with my fix for updating the best tip